### ISSUE TITLE
HTTPSource status is not ready due to Ingress not reconciled

### ISSUE THREAD

**Issue by anishj0shi** [kyma-project/kyma#6945]
### Issue:

When we create an ```httpsource```, sometimes, the status is set **IngressNotConfigured**

On checking networking istio logs from knative-serving namespace:
```json
{
  "level": "error",
  "ts": "2020-01-22T13:01:58.144Z",
  "logger": "istiocontroller.ingress-controller.status-manager",
  "caller": "ingress/status.go:366",
  "msg": "Probing of http://e2e-mesh-c.kyma-integration:80/ failed, IP: 10.44.2.18:80, ready: false, error: unexpected status code: want [200], got 503 (depth: 0)",
  "commit": "6b0e5c6",
  "knative.dev/controller": "ingress-controller",
  "stacktrace": "knative.dev/serving/pkg/reconciler/ingress.(*StatusProber).processWorkItem\n\t/home/prow/go/src/knative.dev/serving/pkg/reconciler/ingress/status.go:366\nknative.dev/serving/pkg/reconciler/ingress.(*StatusProber).Start.func1\n\t/home/prow/go/src/knative.dev/serving/pkg/reconciler/ingress/status.go:268"
}
```

 ```console
$ kubectl get -n kyma-integration ksvc
service.serving.knative.dev/i-might-break-5hg7q   http://i-might-break-5hg7q.kyma-integration.svc.cluster.local   i-might-break-5hg7q-s95hv   i-might-break-5hg7q-s95hv   Unknown   IngressNotConfigured
```

### Findings:
1. Sometimes the pod created by the ksvc does not get a istio-sidecar injected
2. sometimes even with the injected sidecar it does not work -> restarting the cluster-local-gateway seems to solve this

**Comment by anishj0shi** [comment]
also see: https://github.com/kyma-project/kyma/pull/6981

**Comment by k15r** [comment]
Will be fixed by #6981
Closing this issue
